nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -949,13 +955,13 @@ private boolean relevantInitializerMethodOrBlock(
       }
       if (ASTHelpers.getSymbol(methodTree).isStatic()) {
         Set<MethodTree> staticInitializerMethods =
-            class2Entities.get(enclosingClassSymbol(enclosingBlockPath)).staticInitializerMethods();
+            castToNonNull(class2Entities.get(enclosingClassSymbol(enclosingBlockPath)))
```

Comment:
Yes, that is true.  We could of course change this code to do a `containsKey()` check before the `get()`, and then we wouldn't need to use a cast.  But I think if the `containsKey()` check failed, we would just throw an exception anyway since that is an invariant violation.  So the `castToNonNull()` call doesn't seem worse.